packages/react/src/Column/Column.tsx

```diff
@@ -0,0 +1,31 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2024 Gemeente Amsterdam
+ */
+
+import clsx from 'clsx'
+import { forwardRef } from 'react'
+import type { ForwardedRef, HTMLAttributes, PropsWithChildren } from 'react'
+
+type ColumnTag = 'article' | 'div' | 'section'
+type ColumnGap = 'xs' | 'sm' | 'md' | 'lg' | 'xl'
```

Comment:
We can use the more verbose versions here

Reply:
We decided not to do the mapping in the Column component (rendering an `xs` class name for an `extra-small` prop value, etc.) but to do this in CSS.

I’ve updated the prop values and added tokens and a stylesheet.